GCPShardJuggler

```diff
@@ -100,7 +115,7 @@ while true; do
     exit 0
   else
     printf "Deleting the zone as $SHARD shards are not assigned.\n\n"
-    ./gcloud dns managed-zones delete "$ZONE_NAME" --verbosity=none --quiet --project="$PROJECT_ID" > /dev/null 2>&1
+    gcloud dns managed-zones delete "$ZONE_NAME" --verbosity=none --quiet --project="$PROJECT_ID" > /dev/null 2>&1
```

Comment:
I see from a previous PR that this was intended, I can remove it from the PR but I do feel the change would make sense

Reply:
Hey @martinvw, sorry for the late reply.
And yes this indeed is intended as of now, since last I know, gcloud by default isn't universally installed in the System and hence added that to avoid unnecessary confusion.